pages/api/fund.ts

```diff
@@ -16,9 +29,31 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
     }
 
     const captchaToken = req.body["h-captcha-response"]
-    const address = fcl.withPrefix(req.body.address)
+    const address = fcl.withPrefix(req.body.address) || ""
     const token = req.body.token
 
+    if (token === "FUSD") {
+      let hasFUSDVault = false
+      try {
+        hasFUSDVault = await fcl
+          .send([
+            fcl.script(scriptCheckFUSDVault),
+            fcl.args([fcl.arg(address, t.Address)]),
+          ])
+          .then(fcl.decode)
```

Comment:
```suggestion
      try {
        const hasFUSDVault = await fcl
          .send([
            fcl.script(scriptCheckFUSDVault),
            fcl.args([fcl.arg(address, t.Address)]),
          ])
          .then(fcl.decode)
```

I think this can be simplified